src/app/donation-start/donation-start-form/donation-start-form.component.html

```diff
@@ -65,11 +65,10 @@
           <h2 class="b-rt-0 b-m-0 b-bold">Tip Big Give</h2>
           <p *ngIf="!campaign.feePercentage && creditPenceToUse === 0">
             <span *ngIf="!alternateCopy">
-              Big Give doesn't charge platform fees. That means we're only able to continue offering our services thanks to
-              the generosity of donors who leave a tip.
+              Big Give is a registered charity (1136547), and since 2008, thanks to the generosity of people like you who leave a tip, we can continue providing a match funding platform for the charities and causes you care about.
             </span>
             <span *ngIf="alternateCopy">
-              Big Give is a registered charity (1136547), and since 2008, our aim has been to help thousands of charities double the difference of their donations. Thanks to the generosity of people like you who leave a tip, we can continue providing a match funding platform for the charities and causes you care about.
+              Big Give is a registered charity (1136547) and since 2008 has been helping thousands of charities double their donations. Tips left by generous people like you go into our Anchor Match Fund to provide further match funding for our charity partners as well as helping us deliver our charitable mission. Doubling the difference.
```

Comment:
slight nitpick, but should we match the two sides of the 'as well as' more closely in verb form? We've got the bare verb 'go' on one side and the gerund/participle 'helping' on the other.